Makefile

```diff
@@ -11,5 +11,5 @@ publish_packages:
 .PHONY: travis_cron travis_push travis_pull_request travis_api
 travis_cron: all
 travis_push: only_build only_test publish_packages
-travis_pull_request: all
+travis_pull_request: only_build only_test_fast
```

Comment:
this is bringing this repo in line with our other ones.  Specifically, it is enabling us to have cron 'full tests' and ci/cd 'short tests'